src/parsing/KeywordInterface.java

```diff
@@ -339,6 +344,14 @@ private static Double arrayIndex(String input) {
 			throw new IllegalArgumentException("The previous output was not a list of numbers.");
 	}
 
+	private static String reset() {
+		clearFunctions();
+		Constant.specialConstants.clear();
+		Constant.specialConstants.put("π", Math.PI);
+		Constant.specialConstants.put("e", Math.E);
```

Comment:
Instead of putting the bulk of this method here, make a static `resetConstants()` in `Constant`. This will reduce dot chaining and allow other UIs to use this functionality.